index.js

```diff
@@ -17,29 +18,14 @@ module.exports = function(el, options) {
     'button',
     '[tabindex]',
   ];
-
-  var candidates = el.querySelectorAll(candidateSelectors);
-
-  if (options.includeContainer) {
-    var matches = Element.prototype.matches || Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
-
-    if (
-      candidateSelectors.some(function(candidateSelector) {
-        return matches.call(el, candidateSelector);
-      })
-    ) {
-      candidates = Array.prototype.slice.apply(candidates);
-      candidates.unshift(el);
-    }
-  }
+  var candidates = deepQuerySelectorAll(el, candidateSelectors, options.includeContainer);
 
   var candidate, candidateIndex;
-  for (var i = 0, l = candidates.length; i < l; i++) {
+  for (var i = 0; i < candidates.length; i++) {
```

Comment:
Please remove this unrelated and unnecessary change. The same goes for the space change below.